hibernate-core/src/main/java/org/hibernate/engine/spi/FilterDefinition.java

```diff
@@ -32,18 +34,25 @@ public class FilterDefinition implements Serializable {
 	private final String filterName;
 	private final String defaultFilterCondition;
 	private final Map<String, JdbcMapping> explicitParamJaMappings = new HashMap<>();
+	private final Map<String, ManagedBean<? extends  Supplier>> parameterResolverMap = new HashMap<>();
+	private final boolean autoEnabled;
 
 	/**
 	 * Construct a new FilterDefinition instance.
 	 *
 	 * @param name The name of the filter for which this configuration is in effect.
 	 */
-	public FilterDefinition(String name, String defaultCondition, @Nullable Map<String, JdbcMapping> explicitParamJaMappings) {
+	public FilterDefinition(String name, String defaultCondition, @Nullable Map<String, JdbcMapping> explicitParamJaMappings,
```

Comment:
Please re-add a construct `public FilterDefinition(String name, String defaultCondition, @Nullable Map<String, JdbcMapping> explicitParamJaMappings) {` that delegates to this constructor, additionally passing `Collections.emptyMap(), false`